functests/1_performance/rt-kernel.go

```diff
@@ -30,33 +32,43 @@ var _ = Describe("[performance]RT Kernel", func() {
 
 	It("[test_id:26861][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] should have RT kernel enabled", func() {
 
-		Eventually(func() string {
+		profile := &performancev1alpha1.PerformanceProfile{}
+		key := types.NamespacedName{
+			Name: testutils.PerformanceProfileName,
+		}
+		err := testclient.Client.Get(context.TODO(), key, profile)
```

Comment:
I think the profile query should not be part of the test, each test may be asking for the profile. If some tests do not need the profile, but is not an issue